### ISSUE TITLE
emovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.

### ISSUE THREAD

**Issue by progval** [gordon-/atable#16]
When running almost any subcommand of manage.py (eg. runserver):

```
/usr/lib/python3.4/importlib/_bootstrap.py:321: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  return f(*args, **kwds)
```

**Comment by opi** [comment]
I guess we don't really care right know, as we run Django 1.8

**Comment by vegaelle** [comment]
That’s not a good answer. We could have reasons to migrate to 1.9 (and I have a few in mind). But IIRC, those errors come from Django itself, so I guess it will be updated with Django

**Comment by vegaelle** [comment]
FIY, a blind-upgrade to Django-1.10.2 solves this issue. Commit is on its way.

**Comment by progval** [comment]
~o~

**Comment by vegaelle** [comment]
Fixed in dc31895.
